api/src/test/java/io/opencensus/stats/NoopViewManagerTest.java

```diff
@@ -105,8 +102,6 @@ public void noopViewManager_GetView_Cumulative() {
     ViewData viewData = viewManager.getView(VIEW_NAME);
     assertThat(viewData.getView()).isEqualTo(view);
     assertThat(viewData.getAggregationMap()).isEmpty();
-    assertThat(viewData.getWindowData())
-        .isEqualTo(CumulativeData.create(Timestamp.create(0, 0), Timestamp.create(0, 0)));
```

Comment:
Done, though NoopStats will no longer return IntervalData.

Reply:
Can we create the no-op ViewData using the old factory method, so that getWindowData() continues to return the expected window data type until it is removed?  I'd rather not change the behavior of the deprecated methods, if possible.  I also think we should continue to test getWindowData for the interval ViewData, whether it returns a cumulative or interval WindowData.

Reply:
SGTM, I've reverted the changes to NoopStats and it will continue to return the same WindowData as before.